templates/index.html

```diff
@@ -1,692 +1,425 @@
+        <div class="col-12 p-logo-links">
+          <a class="p-logo-link p-card" href="https://docs.snapcraft.io/build-snaps/python">
+            <header class="p-card__header">
+              <img src="https://assets.ubuntu.com/v1/c3d9d13f-python-logo.png" style="height: 105px; max-height: 105px;" alt=""/>
+            </header>
+            <p class="u-no-margin">Python</p>
+          </a>
+          <a class="p-logo-link p-card" href="https://docs.snapcraft.io/build-snaps/node">
+            <header class="p-card__header">
+              <img src="https://assets.ubuntu.com/v1/9735ad74-node-logo.png" style="height: 105px; max-height: 105px;" alt=""/>
+            </header>
+            <p class="u-no-margin">Node.js</p>
+          </a>
+          <a class="p-logo-link p-card" href="https://docs.snapcraft.io/build-snaps/go">
+            <header class="p-card__header">
+              <img src="https://assets.ubuntu.com/v1/c85a212e-go-logo.png" style="height: 105px; max-height: 105px;" alt=""/>
+            </header>
+            <p class="u-no-margin">Go</p>
+          </a>
+          <a class="p-logo-link p-card" href="https://docs.snapcraft.io/build-snaps/moos">
+            <header class="p-card__header">
+              <img src="https://assets.ubuntu.com/v1/04ff3e39-MOOSV-10-256.jpg" style="height: 105px; max-height: 105px;" alt=""/>
+            </header>
+            <p class="u-no-margin">MOOS</p>
+          </a>
+          <a class="p-logo-link p-card" href="https://docs.snapcraft.io/build-snaps/ros">
+            <header class="p-card__header">
+              <img src="https://assets.ubuntu.com/v1/dc84f68e-c8749268-logo-ros.png" style="height: 105px; max-height: 105px;" alt=""/>
+            </header>
+            <p class="u-no-margin">ROS</p>
+          </a>
+          <a class="p-logo-link p-card" href="https://docs.snapcraft.io/build-snaps/pre-built">
+            <header class="p-card__header">
+              <img src="https://assets.ubuntu.com/v1/f14812c7-adwaita-package.png" style="height: 105px; max-height: 105px;" alt=""/>
+            </header>
+            <p class="u-no-margin">Pre-built apps</p>
+          </a>
+        </div>
+      </div>
+    </div>
 
-                <p>Typically, configuration is stored in one of these, along with
-                system-wide data for the snap.</p>
+    <div class="p-strip is-shallow is-bordered u-no-padding--bottom">
+      <div class="row u-vertically-center">
+        <div class="col-6">
+          <h2>Auto-build from GitHub</h2>
+          <p class="p-heading--five">Automate your build process and publish software straight from your GitHub repos.</p>
+        <p><a class="p-button--neutral" href="https://build.snapcraft.io/auth/authenticate"><i class="p-icon--github"></i> Get started with GitHub</a></p>
+        </div>
+        <div class="col-6">
+          <img class="u-float-right" src="https://assets.ubuntu.com/v1/006db1f4-build-snapcraft-heroku.png" />
```

Comment:
Instead of using `u-no-padding--bottom` please use the `image-position` pattern: https://docs.vanillaframework.io/en/utilities/image-position

Reply:
Please add `alt` attr